src/Evenement/EventEmitterTrait.php

```diff
@@ -31,7 +31,7 @@ public function once($event, callable $listener)
         $onceListener = function () use (&$onceListener, $event, $listener) {
             $this->removeListener($event, $onceListener);
 
-            call_user_func_array($listener, func_get_args());
+            \call_user_func_array($listener, \func_get_args());
```

Comment:
You could add the `...$args` here, too, but then you have to bump the minimum version.